docs/_layouts/docs.html

```diff
@@ -3,193 +3,227 @@
-          IP-per-task
-        </a>
-      </li>
-      <li>
-        <a href="{{ site.baseurl }}/docs/marathon-ui.html">
-          Marathon UI
-        </a>
-      </li>
-      <li>
-        <a href="{{ site.baseurl }}/docs/metrics.html">
-          Metrics
-        </a>
-      </li>
-      <li>
-        <a href="{{ site.baseurl }}/docs/plugin.html">
-          Plugins
-        </a>
-      </li>
-      <li>
-        <a href="{{ site.baseurl }}/docs/ports.html">
-          Ports
-        </a>
-      </li>
-      <li>
-        <a href="{{ site.baseurl }}/docs/native-docker-private-registry.html">
-          Private Docker Registries
-        </a>
-      </li>
-      <li>
-        <a href="{{ site.baseurl }}/docs/recipes.html">
-          Recipes
-        </a>
-      </li>
-      <li>
-        <a href="{{ site.baseurl }}/docs/generated/api.html">
-          REST API
-        </a>
-      </li>
-      <li>
-        <a href="{{ site.baseurl }}/docs/rest-api.html">
-          REST API (deprecated)
-        </a>
-      </li>
-      <li>
-        <a href="{{ site.baseurl }}/docs/setup.html">
-          Setup Recommendations for Marathon
-        </a>
-      </li>
-      <li>
-        <a href="{{ site.baseurl }}/docs/task-environment-vars.html">
-          Task Environment Variables
-        </a>
-      </li>
-      <li>
-        <a href="{{ site.baseurl }}/docs/troubleshooting.html">
-          Troubleshooting
-        </a>
-      </li>
-    </ul>
-    <hr>
-    <h5>Development</h5>
-    <ul class="list-unstyled">
-      <li>
-        <a href="{{ site.baseurl }}/docs/deployment-design-doc.html">
-          Deployment Design
-        </a>
-      </li>
-      <li>
-        <a href="{{ site.baseurl }}/docs/core-architecture.html">
-          New Core Architecture
-        </a>
-      </li>
-    </ul>
-    <hr>
-    <h5>Upgrading</h5>
-    <ul class="list-unstyled">
-      <li>
-        <a href="{{ site.baseurl }}/docs/upgrade/index.html">
-          Upgrading to a Newer Version
-        </a>
-      </li>
-      <li>
-        <a href="{{ site.baseurl }}/docs/upgrade/06xto070.html">
-          0.6.x to 0.7.0
-        </a>
-      </li>
-    </ul>
-  </div>
-  <div class="col-sm-9">
-    {{ content }}
-  </div>
-</div>
+    <div class="col-sm-3">
+
+        <!-- <h5>Overview: What is Marathon and why is useful?</h5> -->
+
+        <h5>Getting Started</h5>
+        <ul class="list-unstyled">
+            <li>
+                <a href="{{ site.baseurl }}/docs/">
+                    Install Marathon
+                </a>
+            </li>
+            <li>
+                <a href="{{ site.baseurl }}/docs/setup.html">
+                    Setup Recommendations
+                </a>
+            </li>
+            <li>
+                <a href="{{ site.baseurl }}/docs/high-availability.html">
+                    High Availability Mode
+                </a>
+            </li>
+        </ul>
+
+        <h5>Application Deployment</h5>
```

Comment:
Problem: _Application Deployment_ is a rather abstract term. All the pages in this section are/should be about using Marathon, so I guess _Using Marathon_ would be a suitable header. Does that make sense to you?